src/Spatial.Tests/Euclidean/LineSegment2DTests.cs

```diff
@@ -139,6 +139,19 @@ public void IntersectWithTest(string s1, string e1, string s2, string e2, string
             Assert.AreEqual(e, result);
         }
 
+        [TestCase("0,0", "-2,-2", "1,0", "1,2", "1,1")]
```

Comment:
Why not add the test cases to the existing test?

Reply:
Why not? One is for positive result, another is for negative.